backend/migrations/000018_add_achievements_table.down.sql

```diff
@@ -0,0 +1 @@
+DROP TABLE IF EXISTS community_users_achievements; 
```

Comment:
Can you also `DROP TYPE IF EXISTS achievement_types` here?

Reply:
Done.